src/darwin/Framework/CHIP/templates/CHIPCommandPayloadsObjc-src.zapt

```diff
@@ -18,6 +18,13 @@ NS_ASSUME_NONNULL_BEGIN
   }
   return self;
 }
+
+- (NSString *)description
+{
+  NSString *descriptionString = [NSString stringWithFormat:@"<CHIP{{asUpperCamelCase parent.name}}Cluster{{asUpperCamelCase name}}Params: {{#zcl_command_arguments}}_{{asStructPropertyName label}}: %@; {{/zcl_command_arguments}}>"{{#zcl_command_arguments}},{{#if isArray}}_{{asStructPropertyName label}}{{else if (isOctetString type)}}[_{{asStructPropertyName label}} base64EncodedStringWithOptions:0]{{else}}_{{asStructPropertyName label}}{{/if}}{{/zcl_command_arguments}}];
```

Comment:
This could _really_ benefit from some linebreaks for readability of the template, outside of the string literals.

Does adding linebreaks into the description string as well make sense?

I just tried this with `darwin-framework-tool accesscontrol read acl ${NODE_ID} 0 --fabric-filtered false` and I get
```
AccessControl.Acl response (
    "<CHIPAccessControlClusterAccessControlEntry: _privilege: 5; _authMode: 2; _subjects: (\n    1351986286021876534\n); _targets: (null); _fabricIndex: 1; >",
    "<CHIPAccessControlClusterAccessControlEntry: _privilege: 0; _authMode: 0; _subjects: (null); _targets: (null); _fabricIndex: 2; >"
)
```

complete with the `\n` like that, which is a little surprising....

Reply:
Why do we want the `_` in the "label" part of the print format?  This part: `{{#zcl_command_arguments}}_{{asStructPropertyName label}}: %@;`

Is that the platform convention?

Reply:
No, it is not platform convention. I will remove the `_`.

Reply:
@bzbarsky-apple I tried to add line breaks in the ZAP file, but it changes the format of the objective-c code.